src/components/NavBar.astro

```diff
@@ -53,8 +53,8 @@ import RegistrationButton from './RegistrationButton.astro';
       flex: 1;
       justify-content: flex-end;
       order: 3;
-      @media (min-width: 640px) {
-        order: 3;
+      @media (max-width: 640px) {
+        display: none;
```

Comment:
This makes a ton of sense on the homepage, where there's already a second CTA in the header. On other pages that don't have a second CTA, this would cause a [WCAG Success Criterion 1.4.10 Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) violation, as zooming in would cause a loss functionality. (Homepage not an issue because the second CTA remains visible, so no on-page functionality is lost)

**Possible options:**
* Only `display: none` the navbar CTA on the homepage
* Provide a hamburger menu on mobile
* Place a second CTA on non-homepage pages that only appears at mobile widths

Of these, I think I like the third option as the simplest implementation.

Reply:
Oh right, totally forgot about the Subpages.
Personal Preference would be Hamburger, but i'd ask @a-trost for feedback on this too.

Reply:
I didn't initially do a hamburger/nav drawer since it was only a few links, seemed sort of unnecessary. 

I like @BenDMyers third idea though! @a-trost what's your thoughts?